app/styles/ember-appuniversum/_c-help-text.scss

```diff
@@ -8,6 +8,7 @@
 $au-help-text-color             : var(--au-gray-800) !default;
 $au-help-text-secondary-color   : var(--au-gray-600) !default;
 $au-help-text-tertiary-color    : var(--au-gray-600) !default;
+$au-help-text-warning-color     : var(--au-yellow-700) !default;
```

Comment:
It seems this color wasn't added to appuniversum yet: https://github.com/appuniversum/appuniversum/blob/cfcfc5536266d2f4c3378e6454bd3f3a98691e38/appuniversum/_s-root.scss#L26-L27
The helptext color isn't changed at the moment :smile:.